examples/model_card_output/template/html/compare.html.jinja

```diff
@@ -1,186 +1,186 @@
-</div>
-{% endmacro %}
-{% macro render_common_reports(common_reports) %}
-{% for _tuple in common_reports %}
- <div style="width: 100%; overflow: hidden;">
-    {# tuple of (report A, report B) #}
-     <div style="width: 50%; float: left;"> {{ render_report(_tuple[0]) }} </div>
-     <div style="margin-left: 50%;"> {{ render_report(_tuple[1]) }} </div>
-</div>
-{% endfor %}
-{% endmacro %}
-{% macro render_test(test) %}
-<div class="col card">
-  {% if test.name %}
-  <h4> {{ test.name  }} </h4>
-  {% endif %}
-  {% if test.description %}
-  <div>
-    <b> Description: </b>
-    {{ test.description }} 
-  </div>
-  {% endif %}
-  {% if test.threshold %}
-  <div>
-    <b> Threshold: </b>
-    {{ test.threshold }} 
-  </div>
-  {% endif %}
-  {% if test.result %}
-  <div>
-    <b> Result: </b>
-    {# looks like a csv file #}
-    {% if '\n' in test.result and ',' in test.result %}
-      {{ render_results_table(test.result) }}
-    {% else %}
-      {{ test.result }} 
-    {% endif %}
-  </div>
-  {% endif %}
-  {% if test.passed == true %} 
-    <div><b style="color:green;"> Passed</b></div> 
-    {% else %}
-    <div><b style="color:red;"> Failed</b></div>
-  {% endif %}
-  {% if test.graphics.collection %}
-    {{ render_metrics_graphics(test.graphics) }}
-  {% endif %}
-</div>
-{% endmacro %}
-{% macro render_results_table(results) %}
-<table>
-{% set results_list = results.split('\n') %}
-  {% for row in results_list %}
-    {% if (row is defined) and row %}
-      <tr>
-      {% set items = row.split(',') %}
-      {% for item in items %}
-      <td>{{ item }}</td>
-      {% endfor %}
-      </tr>
-    {% endif %}
-  {% endfor %}
-</table>
-{% endmacro %}
-
-<html lang="en">
-<head>
-<style>
-  h1 {text-align: left;}
-  .row {
-    display: flex;
-  }
-  .col {
-    flex: 1;
-    align-items: center; 
-    justify-content: center;
-  }
-  .card {
-    padding: 1em;
-    border: 1px solid #DADCE0;
-    margin: 10px;
-  }
-  .card .card {
-    padding: 1em;
-    border: none;
-    margin: 10px;
-  }
-  .img-container {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-around;
-    text-align: center;
-  }
-  .img-item {
-    flex: 1;
-  }
-  img {
-    max-width: 100%
-  }
-  .center {
-    margin-left: auto;
-    margin-right: auto;
-    text-align: center;
-  }
-  table, th, td { border: 1px solid black; }
-  th, td { border: 1px solid #CCC; height: 30px; }
-  caption { font-weight: bold; }
-</style>
-<title>Model Card Comparison</title>
-</head>
-<body>
-{{ render_header(card_a_name, card_b_name) }}
-{% if common_reports_pm %}
-    <h2>Performance Metrics</h3>
-    {{ render_common_reports(common_reports_pm) }}
-{% endif %}
-{% if common_reports_er %}
-    <h2>Explainability Reports</h3>
-    {{ render_common_reports(common_reports_er) }}
-{% endif %}
-{% if common_reports_fr %}
-    <h2>Fairness Reports</h3>
-    {{ render_common_reports(common_reports_fr) }}
-{% endif %}
-</body>
-</html>
+<!DOCTYPE html>


```

Comment:
Wait a minute, I see a lot of `^M` character which is supposedly a text return character used in some windows editors. Can you configure your text editor to use UNIX newline return and push the clean template. Thanks